google-cloud-pubsub/support/doctest_helper.rb

```diff
@@ -416,6 +416,20 @@ def mock_pubsub
     end
   end
 
+  doctest.skip "Google::Cloud::Pubsub::Topic#publish_async" do
```

Comment:
OOC why is publish_async skipped? Subscription#listen has a useful code comment describing why it's being skipped.